From 61bca0c7b3dd91fd3000f9264d63162adf6f7787 Mon Sep 17 00:00:00 2001
From: Kill3rEz <98993017+Kill3rEz@users.noreply.github.com>
Date: Fri, 5 Jun 2026 18:49:06 +0200
Subject: [PATCH] Increase BINDER_VM_SIZE from 1MB to 4MB

Increase BINDER_VM_SIZE from 1MB to 4MB for fix RAW processing on Master Mode

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>

diff --git a/libs/binder/ProcessState.cpp b/libs/binder/ProcessState.cpp
index f7d53472a552afa390d3f679310cfa70568a85b3..8af14bd30fd58f8a8d02248be6a8a537478a8efd 100644
--- a/libs/binder/ProcessState.cpp
+++ b/libs/binder/ProcessState.cpp
@@ -45,7 +45,7 @@
 #include <unistd.h>
 #include <mutex>
 
-#define BINDER_VM_SIZE ((1 * 1024 * 1024) - sysconf(_SC_PAGE_SIZE) * 2)
+#define BINDER_VM_SIZE ((4 * 1024 * 1024) - sysconf(_SC_PAGE_SIZE) * 2)
 #define DEFAULT_MAX_BINDER_THREADS 15
 #define DEFAULT_ENABLE_ONEWAY_SPAM_DETECTION 1
 
